more_itertools/more.py

```diff
@@ -5137,6 +5137,8 @@ def nth_prime(n, *, approximate=False):
     >>> nth_prime(200_000_000, approximate=True)  # Exact result is 4222234763
     4217820427
 
+    In the future, more accurate approximations may be returned.
```

Comment:
I'd rather leave this statement out - if and when the state of the art in mathematics research advances, we can implement the changes. But for now things are the way they are.